zombies.js

```diff
@@ -310,15 +426,20 @@
  * @param {number} speed            The zombie's speed.
  */
 
+function ExplodingZombie(health, strength, speed) {
+  Zombie.call(this, health, strength, speed);
+}
+ExplodingZombie.prototype = Object.create(Zombie.prototype, {
+  constructor: {
+    value: Zombie
+  }
+});
```

Comment:
This should follow structure: `DeathMetalSong.prototype = Object.create(MetalSong.prototype);`